plugin.video.embycon/service.py

```diff
@@ -289,11 +289,13 @@ def onPlayBackSeek(self, time, seekOffset):
 monitor = Service()
 last_progress_update = time.time()
 download_utils.checkVersion()
+home_window = HomeWindow()
 
-xbmc_monitor = xbmc.Monitor()
-while not xbmc_monitor.abortRequested():
+# monitor.abortRequested() is causes issues, it currently triggers for all addon cancelations which causes
```

Comment:
Three is one, and in fact it has been fixed in 18 but not backported yet
https://forum.kodi.tv/showthread.php?tid=319161